### DISCUSSION TITLE
Error on mac

### DISCUSSION THREAD

**Discussion by DrEinsteinCode** [Flam3rboy/discord-bot-client#221] [Q&A]
i tried instaling the zip and the dmg and it showed me this.
![Screen Shot 2021-01-27 at 10 39 58 AM](https://user-images.githubusercontent.com/73256469/106023007-04f84700-608c-11eb-9511-a49fe18322fe.png) pls help

**Comment by samuelscheit** [comment]
duplicate of #165
